spec/support/features.rb

```diff
@@ -1,6 +1,5 @@
 # spec/support/features.rb
 require File.expand_path('../features/session_helpers', __FILE__)
-require File.expand_path('../fixture_helpers', __FILE__)
```

Comment:
@val99erie I'm guessing we might still want to test the different thumbnail types in the future.  Right now a work has no thumbnail, but #1073 adds them back in.